src/controllers/petsController.js

```diff
@@ -1,8 +1,10 @@
 const petsModel = require('../models/petsModel.js');
 const { inputValidation } = require("../utils/tools.js");
-const { ContentTypeError, PropNullorEmptyError, PropRequiredError } = require("../utils/errors.js");
+const { ContentTypeError, PropNullorEmptyError, PropRequiredError, AuthorizationError } = require("../utils/errors.js");
```

Comment:
is `AuthorizationError` used in this file?

Reply:
It was initially, but not anymore. Same with the jwt and SECRET variables. I'll remove them!